contracts/compliance/modular/ModularCompliance.sol

```diff
@@ -60,24 +60,41 @@
  *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-pragma solidity 0.8.17;
+pragma solidity 0.8.26;
 
 import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import "../../token/IToken.sol";
 import "./IModularCompliance.sol";
 import "./MCStorage.sol";
 import "./modules/IModule.sol";
+import "../../libraries/errors/InvalidArgumentErrors.sol";
 
 
 contract ModularCompliance is IModularCompliance, OwnableUpgradeable, MCStorage {
 
+    /// errors
+
+    error AddressNotATokenBoundToComplianceContract();
+
+    error CannotAddMoreThan25Modules();
+
+    error ComplianceNotSuitableForBindingToModule();
+
+    error ModuleAlreadyBound();
```

Comment:
some of these errors should be imported from the compliance errors library i guess, these are very common to modules and compliance contracts